gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/src/main/java/io/gravitee/gateway/standalone/vertx/VertxReactorConfiguration.java

```diff
@@ -58,14 +65,14 @@ public ServerManager serverManager(
             while ((environment.getProperty(prefix + ".type")) != null) {
                 final VertxServerOptions options = VertxServerOptions
                     .builder(environment, prefix, keyStoreLoaderManager)
-                    .defaultPort(8082)
+                    .defaultPort(Objects.equals(environment.getProperty("%s.type".formatted(prefix)), TCP_PREFIX) ? TCP_DEFAULT_PORT : 8082)
                     .build();
                 serverManager.register(serverFactory.create(options));
                 prefix = SERVERS_PREFIX + "[" + counter++ + "]";
             }
         } else {
             // No server list configured, fallback to single 'http' server configuration.
-            final VertxHttpServerOptions options = VertxHttpServerOptions
+            final VertxHttpServerOptions httpOptions = VertxHttpServerOptions
```

Comment:
should we verify if http is enabled ? as you do for tcp.
maybe we would want to only have tcp as a fallback ?

Reply:
@jhaeyaert any opinion regarding this ?

Reply:
I don't know yet if completely disabling http is something that will be expected in the end. I don't think it matters for now. We can decide later if we need to introduce this `http.enabled` option or not.